diff -up icecast-2.4.4/conf/icecast.xml.in.orig icecast-2.4.4/conf/icecast.xml.in
--- icecast-2.4.4/conf/icecast.xml.in.orig	2018-10-31 18:52:49.000000000 +0100
+++ icecast-2.4.4/conf/icecast.xml.in	2022-09-27 00:25:22.097945924 +0200
@@ -161,7 +161,7 @@
         <hidden>1</hidden>
         <public>1</public>
         <authentication type="htpasswd">
-                <option name="filename" value="myauth"/>
+                <option name="filename" value="/usr/share/icecast/myauth"/>
                 <option name="allow_duplicate_users" value="0"/>
         </authentication>
         <http-headers>
@@ -195,10 +195,10 @@
 
         <!-- Note that if <chroot> is turned on below, these paths must both
              be relative to the new root, not the original root -->
-        <logdir>@localstatedir@/log/@PACKAGE@</logdir>
+        <logdir>/var/log/icecast</logdir>
         <webroot>@pkgdatadir@/web</webroot>
         <adminroot>@pkgdatadir@/admin</adminroot>
-        <!-- <pidfile>@pkgdatadir@/icecast.pid</pidfile> -->
+        <pidfile>/var/run/icecast/icecast.pid</pidfile>
 
         <!-- Aliases: treat requests for 'source' path as being for 'dest' path
              May be made specific to a port or bound address using the "port"
@@ -234,10 +234,10 @@
 
     <security>
         <chroot>0</chroot>
-        <!--
+        <!-- Changing user will break logrotate support ( cannot create pid file )
         <changeowner>
-            <user>nobody</user>
-            <group>nogroup</group>
+            <user>icecast</user>
+            <group>icecast</group>
         </changeowner>
         -->
     </security>
